inc/sitemaps/class-sitemaps.php

```diff
@@ -419,9 +419,15 @@ public function output() {
 	}
 
 	/**
-	 * Make a request for the sitemap index so as to cache it before the arrival of the search engines.
+	 * Makes a request for the sitemap index so as to cache it before the arrival of the search engines.
```

Comment:
`Makes a request to the sitemap index to cache it before the arrival of the search engines.` makes for a better sentence.